apps/dashboard/src/app/components/messages/message-modal.component.html

```diff
@@ -0,0 +1,8 @@
+<pa-modal-dialog>
+  <pa-modal-title>{{ data?.title | translate }}</pa-modal-title>
+  <pa-modal-content [innerHtml]="data?.['message']"></pa-modal-content>
+  <pa-modal-footer>
+    <pa-button kind="primary"
+               (click)="modal.close()">{{ 'pastanaga.confirm' | translate}}</pa-button>
```

Comment:
On the previous implementation, the label was "ok". Does changing it to "Confirm" makes sense?